### ISSUE TITLE
Regression: Can't upload files after pull request #4862 was merged

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#5194]
Tested on https://dev1.dataverse.org running e948af2 on the "develop" branch.

![screen shot 2018-10-15 at 2 39 15 pm](https://user-images.githubusercontent.com/21006/46970838-54b3cf80-d088-11e8-998f-f40c4f95927b.png)

![screen shot 2018-10-15 at 2 39 27 pm](https://user-images.githubusercontent.com/21006/46970837-54b3cf80-d088-11e8-8dd9-a55d8c969cd6.png)

**Comment by landreev** [comment]
@kcondon  the fix is in the db update script.

**Comment by kcondon** [comment]
@landreev @pdurbin 
We spoke briefly and think that maybe this should check whether the value is already set and then depending not set it.

**Comment by pdurbin** [comment]
Yeah, @sekmiller and I talked about that too but that would really only going to affect @pameyer so we can just let him know.
